### PR TITLE
Implemented task and tests

### PR DESCRIPTION
**Pull request by ekreed**
- [DEMO LINK](https://ekreed.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://ekreed.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,82 @@
...
+              >Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#samsung"
+              >Samsung
+            </a>
+          </li>
```
Comment (nazarmatsevych) [inline src/index.html:39]:
```suggestion
          <li class="nav__item">
            <a
              class="nav__link is-active"
              href="#apple"
            >
              Apple
            </a>
          </li>
          
          <li class="nav__item">
            <a
              class="nav__link"
              href="#samsung"
            >
              Samsung
            </a>
          </li>
```

Add blank line betwen multiple sibling blocks of html and fix formatting

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,69 @@
+
+html {
+  --blue-color: #00acdc;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
```
Comment (nazarmatsevych) [inline src/style.css:5]: Move box-sizing to * selector

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 2 above)
Comment (ekreed) [reply src/style.css:5]: У чек листу було вказано не використовувати селектор *

File: src/index.html (hunk 1 above)
Comment (ekreed) [reply src/index.html:39]: done

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
No need to delete the file index.html. It needs to be returned

**Pull request closed without merging**